lib/ui.rb

```diff
@@ -30,20 +30,30 @@ def continue?
     get_continue_request
   end
 
-  def get_contact_details(fields)
-    new_contact = {}
-    output.puts "#{CLEAR_SCREEN}"
-    fields.each do |field_name|
-      new_contact[field_name] = details_for(field_name)
-    end
-    new_contact
+  def details_for(field_name)
+    output.print "#{format_for_display(field_name)}: "
+    get_data_for_field(field_name)
+  end
+
+  def ask_for_search_keyword(keyword)
+    output.puts "\nEnter #{format_for_display(keyword).downcase}:"
+    get_search_keyword(keyword)
+  end
+
+  def confirm_contact_creation
+    output.puts "\n\nContact created!"
   end
 
   private
 
-  def details_for(field_name)
-    output.print "#{format_for_display(field_name)}: "
-    get_data_for_field(field_name)
+  def get_search_keyword(keyword)
+    user_input = input.gets.chomp
+    if user_input == ""
+      output.puts "#{CLEAR_SCREEN}\nPlease provide the search keyword: "
```

Comment:
If you put a small abstraction around `output.puts` you won't need to write (and test for!) `CLEAR_SCREEN` in multiple places.